LICENSE

```diff
@@ -0,0 +1,24 @@
+MIT License
```

Comment:
I think it's better that @Eomm add this, he authored the site after all

Reply:
It is fine 👍🏼

Reply:
np! removed file to keep scope of PR to a single change